src/Algebra/Graph/Undirected.hs

```diff
@@ -179,7 +180,7 @@ instance (Show a, Ord a) => Show (Graph a) where
 {- See Note [Functions for rewrite rules] in 'Algebra.Graph' -}
 
 instance Functor Graph where
-    fmap f = UG . fmap f . fromUndirected
+    fmap f = UG . fmap f . _fromUndirected
```

Comment:
I think we probably don't need `_fromUndirected` at all, and should use `coerce` here and in `>>=`.

Reply:
You're right!